packages/syft/src/syft/service/dataset/dataset.py

```diff
@@ -138,7 +138,9 @@ class Asset(SyftObject):
     __repr_attrs__ = ["name", "shape"]
 
     def __init__(
-        self, description: Optional[Union[MarkdownDescription, str]] = "", **data
+        self,
+        description: Optional[Union[MarkdownDescription, str]] = "",
+        **data: Dict[str, Any],
```

Comment:
```suggestion
        **data: Any,
```

`*args` and `**kwargs` should be typed as `*args: Any` and `**kwargs: Any`, not `*args: Tuple[Any]` and `**kwargs: Dict[str, Any]`. see #6791